app/src/main/res/values/strings.xml

```diff
@@ -237,6 +237,7 @@
   <string name="transaction_type_poa_offchain">@string/transaction_type_poa</string>
   <string name="transaction_type_standard">Transfer</string>
   <string name="transaction_type_bonus">Bonus</string>
+  <string name="transaction_type_subscription">Subscription</string>
```

Comment:
I think this is not being used

Reply:
Be careful when removing strings (or adding) to avoid conflicts with the strings set by the copy team.

Reply:
This is being used in **TransactionDetailActivity** but it should be removed when the new screens are ready